Events.php

```diff
@@ -34,11 +34,14 @@ public static function onAdminMenuInit($event)
     }
 
     public static function addAdFrame($event)
-    {
-        if (Yii::$app->user->isGuest || Yii::$app->user->identity && AuthHelper::isGuestAccessEnabled())
+    {	 
+        if (
+			Yii::$app->user->isGuest && 
```

Comment:
From reviewing the code itself, I see that this will only display ads to guests and not those that are logged in, am I correct in this?